### ISSUE TITLE
dashboardTime is not saved

### ISSUE THREAD

**Issue by olivierlambert** [influxdata/chronograf#1502]
Hi lads!

When creating a new visualization in Chronograf (1.3 from .deb), the `:dashboardTime:` is not always saved correctly, and often replaced by `now() - 15m`.

Eg, I'm creating a new panel in dashboard, copy/pasting this request:

```
SELECT derivative("packets_recv",1s)*-1, derivative("packets_sent",1s) FROM "telegraf"."autogen"."net" WHERE time > :dashboardTime: GROUP BY "host",* fill(0)
```
As soon as I validate the request, it's automatically replaced by:

```
SELECT derivative("packets_recv",1s)*-1, derivative("packets_sent",1s) FROM "telegraf"."autogen"."net" WHERE time > now() - 15m GROUP BY "host"
```

Note: I can replace the `now() -15m` by `now() -1h`, it works. But obviously, the dashboard timepicker is then useless because the graph won't change the timescale.

Am I missing something here?

**Comment by nhaugo** [comment]
I am fairly certain this is related to https://github.com/influxdata/chronograf/issues/1482

That issue mention "remove hack..." I think that hack is causing this issue, I will see if there is a temporary workaround to get this fixed in 1.3.1

**Comment by olivierlambert** [comment]
Thanks for your answer @nhaugo :+1: I'll keep an eye on the next release and confirm you if it's solved by the coming fix. Keep up the good work!

**Comment by olivierlambert** [comment]
I confirm this is fixed in `1.3.1` :+1:
